repair/featurize/initattrfeat.py

```diff
@@ -10,20 +10,42 @@ def gen_feat_tensor(input, classes, total_attrs):
     vid = int(input[0])
     attr_idx = input[1]
     init_idx = int(input[2])
-    tensor = -1 * torch.ones(1,classes,total_attrs)
+    tensor = -1 * torch.ones(1, classes, total_attrs)
     tensor[0][init_idx][attr_idx] = 1.0
     return tensor
 
 
 class InitAttrFeaturizer(Featurizer):
+
+    def __init__(self, init_weight=1.0):
+        """
+        InitAttrFeaturizer cannot be learnable.
+
+        :param init_weight: (float or list of floats) a fixed weight for all attributes
+                            or a list of floats that represent the weights of attributes
+                            in the same order in the dataset.
+        """
+        if isinstance(init_weight, list):
+            # If init_weight is a list, we convert to a tensor to be correctly
+            # initialized in the TiedLinear model initialization, where init_weight
+            # is multiplied by a tensor of ones for initialization.
+            init_weight = torch.FloatTensor(init_weight)
+
+        Featurizer.__init__(self, learnable=False, init_weight=init_weight)
```

Comment:
Instead of this featurizer being hard-coded as `learnable=False`, how about just surfacing the `learnable` parameter in `__init__` but make it `false` by default?

Reply:
I think our decision was to **never** make Init learnable, so I did not want to even give the user the choice to.

Reply:
Fair enough 👍